elementary/utils/package.py

```diff
@@ -13,8 +17,19 @@ def get_package_version() -> Optional[str]:
 
 def get_latest_package_version() -> Optional[str]:
     try:
-        resp = requests.get("https://pypi.org/pypi/elementary-data/json")
+        resp = requests.get(_PYPI_URL)
         resp.raise_for_status()
         return resp.json().get("info", {}).get("version")
     except Exception:
         return None
+
+
+def check_dbt_pkg_compatible(dbt_pkg_ver: str):
+    dbt_pkg_ver = version.parse(dbt_pkg_ver)
+    py_pkg_ver = version.parse(get_package_version())
+    if dbt_pkg_ver.major != py_pkg_ver.major or dbt_pkg_ver.minor != py_pkg_ver.minor:
+        click.secho(
+            "You are using incompatible versions between 'edr' and Elementary's dbt package.\n"
+            "Please change the major and minor versions to align.\n",
+            fg="yellow",
```

Comment:
Maybe print that minor and major should be aligned so the user will know what to do?